WalletWasabi/Backend/Models/FilterModel.cs

```diff
@@ -35,19 +35,26 @@ public static FilterModel FromLine(string line)
 	{
 		try
 		{
-			string[] parts = line.Split(':');
+			// Splitting lines using Split(':') requires allocations. Working with .NET spans is faster.
+			ReadOnlySpan<char> span = line;
 
-			if (parts.Length < 5)
+			int m1 = line.IndexOf(':', 0);
+			int m2 = line.IndexOf(':', m1 + 1);
+			int m3 = line.IndexOf(':', m2 + 1);
+			int m4 = line.IndexOf(':', m3 + 1);
```

Comment:
Can we have better namings for these?

For example
`m1` could be `heightIndex`
`m2` could be `blockhashIndex`